src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const basePrice = days * 40;
+  const midleTerm = 3;
+  const longTerm = 7;
+  const lowDiscount = 20;
+  const hiDiscount = 50;
+
+  if (days < midleTerm) {
+    return basePrice;
+  } else if (days >= midleTerm && days < longTerm) {
+    return basePrice - lowDiscount;
+  } else {
+    return basePrice - hiDiscount;
+  }
```

Comment:
let's get rid of if else chain)

```suggestion
  if (days < midleTerm) {
    return basePrice;
  }
  
    if (days >= midleTerm && days < longTerm) {
      return basePrice - lowDiscount;
    } 
  
  
    return basePrice - hiDiscount;
```

Reply:
when we use  return  inside if we don't need to use else after it